static/app/views/dashboardsV2/widgetBuilder/utils.tsx

```diff
@@ -104,18 +108,24 @@ export function normalizeQueries({
   }
 
   if ([DisplayType.TABLE, DisplayType.TOP_N].includes(displayType)) {
-    if (!queries[0].orderby && widgetBuilderNewDesign) {
-      const orderBy = (
-        widgetType === WidgetType.DISCOVER
-          ? generateOrderOptions({
-              widgetType,
-              widgetBuilderNewDesign,
-              columns: queries[0].columns,
-              aggregates: queries[0].aggregates,
-            })[0].value
-          : IssueSortOptions.DATE
-      ) as string;
-      queries[0].orderby = orderBy;
+    if (widgetBuilderNewDesign) {
+      if (!queries[0].orderby) {
+        const orderBy = (
+          widgetType === WidgetType.DISCOVER
+            ? generateOrderOptions({
+                widgetType,
+                widgetBuilderNewDesign,
+                columns: queries[0].columns,
+                aggregates: queries[0].aggregates,
+              })[0].value
+            : IssueSortOptions.DATE
+        ) as string;
+        queries[0].orderby = orderBy;
+      }
+
+      if (!!queries[0].columns.length) {
+        queries[0].columns = [];
+      }
```

Comment:
Mostly a question, should we also be updating `fields`? Since `fields` is supposed to be the aggregates + columns